code_samples/map.js

```diff
@@ -13,7 +13,7 @@ var removePerson = function (map, key) {
 
 var getPerson = function (map, key) {
     return map.get(key).then(function(val) {
-        console.log('Person with id ' + key + ': ' + JSON.stringify(val));
+        console.log('Person with id ' + JSON.stringify(key) + ': ' + JSON.stringify(val));
```

Comment:
Why did you add `JSON.stringify` here? `key` is already a number. There is no need to `stringify` it before printing to console. We `stringify` `val` because it is an `object` and not printable nicely.